### ISSUE TITLE
Feature: accessibility tab order

### ISSUE THREAD

**Issue by m-abs** [Notalib/nativescript-accessibility-ext#12]
We should be able to set the tab ordering for accessible elements in NativeScript

On iOS we have [UIAccessibilityContainer](https://developer.apple.com/documentation/uikit/accessibility/uiaccessibilitycontainer)

On Android the View have these properties:
android:nextFocusUp, android:nextFocusDown, android:nextFocusLeft, and android:nextFocusRight, 
https://developer.android.com/reference/android/view/View.html#attr_android:nextFocusUp
